ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel_correlation.ambr

```diff
@@ -0,0 +1,442 @@
+                        step_0,
+                        latest_0,
+                        step_1,
+                        min(latest_1) over (PARTITION by aggregation_target
+                                            ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_1
+                 FROM
+                   (SELECT aggregation_target,
+                           timestamp,
+                           if(event = 'user signed up', 1, 0) as step_0,
+                           if(step_0 = 1, timestamp, null) as latest_0,
+                           if(event = 'paid', 1, 0) as step_1,
+                           if(step_1 = 1, timestamp, null) as latest_1
+                    FROM
+                      (SELECT e.event as event,
+                              e.team_id as team_id,
+                              e.distinct_id as distinct_id,
+                              e.timestamp as timestamp,
+                              e.$group_0 as aggregation_target
+                       FROM events e
+                       INNER JOIN
+                         (SELECT distinct_id,
+                                 argMax(person_id, _timestamp) as person_id
+                          FROM
+                            (SELECT distinct_id,
+                                    person_id,
+                                    max(_timestamp) as _timestamp
+                             FROM person_distinct_id
+                             WHERE team_id = 2
+                             GROUP BY person_id,
+                                      distinct_id,
+                                      team_id
+                             HAVING max(is_deleted) = 0)
+                          GROUP BY distinct_id) AS pdi ON events.distinct_id = pdi.distinct_id
+                       WHERE team_id = 2
+                         AND event IN ['paid', 'user signed up']
+                         AND timestamp >= '2020-01-01 00:00:00'
+                         AND timestamp <= '2020-01-14 23:59:59' ) events
+                    WHERE (step_0 = 1
+                           OR step_1 = 1) ))
+              WHERE step_0 = 1 SETTINGS allow_experimental_window_functions = 1 ))
+        GROUP BY aggregation_target,
+                 steps
+        HAVING steps = max_steps SETTINGS allow_experimental_window_functions = 1)
+     WHERE steps IN [1, 2]
+     ORDER BY aggregation_target
+     OFFSET 0 SETTINGS allow_experimental_window_functions = 1),
+       toDateTime('2020-01-14 23:59:59') AS date_to,
+       toDateTime('2020-01-01 00:00:00') AS date_from,
+       2 AS target_step,
+       ['user signed up', 'paid'] as funnel_step_names
+  SELECT event.event AS name,
+         countDistinctIf(person.person_id, person.steps = target_step) AS success_count,
+         countDistinctIf(person.person_id, person.steps <> target_step) AS failure_count
+  FROM events AS event
+  JOIN funnel_people AS person ON person.person_id = events.$group_0
```

Comment:
Can I make this clearer here? despite the `funnel_people.person_id` constraint?

Reply:
Hmm, makes things more confusing right now, as means updating table name else where, while keeping the `person_id` variable. So, opting not to do this right now.